driver_scripts/setup_ocean_region_groups.py

```diff
@@ -91,7 +90,7 @@ def remove_small_polygons(inFileName, outFileName, minArea):  # {{{
     write_all_features(features, outFileName, indent=4)  # }}}
 
 
-def build_basins():  # {{{
+def build_ocean_basins():  # {{{
     basinGroupName = 'OceanBasinRegionsGroup'
     basinFileName = 'oceanBasins.geojson'
```

Comment:
The only case in which this would matter is if there are cells that are not land in MPAS-Ocean but also don't fall into any of the basins defined in this repo.  This can happen because the definition of the land currently doesn't exactly match up to the definition of the ocean basins, leaving little gaps in some places.  So the question is, do you want to be sure that the global ocean is really the sum of the regional basins (excluding the overlapping Equatorial Basin) or would you rather be sure that every single cell is included.

> In the hard-wired regions it's implemented as 'take all cells', which I assume means the former. 

Yes, I'm sure the hard-wired regions would be more consistent with the latter, but I don't think we want to use the hard-wired regions as a guide because I think they were more like temporary hacks that were easy to add directly in the code.